README.md

```diff
@@ -53,19 +53,19 @@ php ezpublish/console assets:install --symlink
 
 ## Usage
 
-Place following helper `{{ show_privacy_cookie_banner(%privacy_policy_url%) }}` somewhere in your footer template before body ending tag. Replace `%privacy_policy_url%` with your policy page address.
+Insert the following `{{ show_privacy_cookie_banner(%privacy_policy_url%) }}` helper somewhere in your footer template before body ending tag. Replace `%privacy_policy_url%` parameter with your policy page address.
```

Comment:
+Insert the following `{{ show_privacy_cookie_banner(%privacy_policy_url%) }}` helper somewhere in your footer template before the body ending tag. Replace the `%privacy_policy_url%` parameter with your policy page address.